src/System.Management.Automation/namespaces/RegistryWrapper.cs

```diff
@@ -17,29 +17,31 @@
 
 namespace Microsoft.PowerShell.Commands
 {
+
+#nullable enable
     internal interface IRegistryWrapper
     {
-        void SetValue(string name, object value);
+        void SetValue(string? name, object value);
 
-        void SetValue(string name, object value, RegistryValueKind valueKind);
+        void SetValue(string? name, object value, RegistryValueKind valueKind);
 
         string[] GetValueNames();
 
         void DeleteValue(string name);
 
         string[] GetSubKeyNames();
 
-        IRegistryWrapper CreateSubKey(string subkey);
+        IRegistryWrapper? CreateSubKey(string subkey);
 
-        IRegistryWrapper OpenSubKey(string name, bool writable);
+        IRegistryWrapper? OpenSubKey(string name, bool writable);
 
         void DeleteSubKeyTree(string subkey);
 
-        object GetValue(string name);
+        object? GetValue(string? name);
```

Comment:
Is null returned when the key doesn't exist?

Reply:
From docs: https://docs.microsoft.com/en-us/dotnet/api/microsoft.win32.registrykey.getvalue?view=net-5.0#Microsoft_Win32_RegistryKey_GetValue_System_String_
> Returns
Object
The value associated with name, or null if name is not found.